themes/base.theme.bash

```diff
@@ -216,12 +242,13 @@ function git_prompt_vars {
     fi
   fi
 
+  # no if for gitstatusd here, user extraction is not supported by it
   [[ "${SCM_GIT_SHOW_CURRENT_USER}" == "true" ]] && SCM_BRANCH+="$(git_user_info)"
```

Comment:
What does `git_user_info` provide? Maybe I can add it to gitstatusd.

Reply:
It's basically a call to git config with some magic guessing initials. I guess this isn't worth adding as it's not really a query to the index or filesystem. I would feel comfortable leaving it as it is.
```
function git_user_info {
  # support two or more initials, set by 'git pair' plugin
  SCM_CURRENT_USER=$(git config user.initials | sed 's% %+%')
  # if `user.initials` weren't set, attempt to extract initials from `user.name`
  [[ -z "${SCM_CURRENT_USER}" ]] && SCM_CURRENT_USER=$(printf "%s" $(for word in $(git config user.name | PERLIO=:utf8 perl -pe '$_=lc'); do printf "%s" "${word:0:1}"; done))
  [[ -n "${SCM_CURRENT_USER}" ]] && printf "%s" "$SCM_THEME_CURRENT_USER_PREFFIX$SCM_CURRENT_USER$SCM_THEME_CURRENT_USER_SUFFIX"
}
```